README.md

```diff
@@ -73,11 +73,10 @@ Prerequisites:
 5. Set permissions for service staff members if needed:
    * Create group(s) (via Django admin) and add user(s) to the group
    * Create service permissions for group manually via Django admin or for example:
-     * `docker exec profile-backend python manage.py add_object_permission BERTH
-     VeneAdmin can_view_profiles`
+     * `docker exec profile-backend python manage.py add_object_permission ServiceName GroupName can_view_profiles`
      * where:
-       * `service_type=BERTH`
-       * `group_name=VeneAdmin`
+       * `service=ServiceName`
```

Comment:
This was a strange help text to begin with, but maybe switch the order?
```
ServiceName = The service
GroupName = The name of the group you want to give permissions to
can_view_profiles = The permission
``` 
Or something like that.

Reply:
Improved